man/mysqldumpslow.1

```diff
@@ -1,27 +1,13 @@
 '\" t
 .\"     Title: \fBmysqldumpslow\fR
 .\"    Author: [FIXME: author] [see http://docbook.sf.net/el/author]
-<<<<<<< HEAD
```

Comment:
`mysqldumpslow` man file must not be updated - it contains Percona-specific instructions

Reply:
My mistake. Only dates were changed. Please ignore the comment.